### PR TITLE
Microdata test

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

**Comment by AlbaMontis** [comment]
LGTM

File: tests/testmicrodata.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  // class 'widget-subheader'
+  /** @type {string} */
+  var subheader = widget.querySelector(SUBHEADER_SELECTOR).textContent;
+
+  // Element with ID 'microdata-container' should contain element with CSS
+  // class 'widget-content'.
+  /** @type {Node} */
+  var content = widget.querySelector(CONTENT_SELECTOR);
+  /** @type {Node} */
+  var list = content.getElementsByTagName('ul');
```
Comment (mshchygol) [inline tests/testmicrodata.js]: Are you shore that type of this variable is {Node} ?  
https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByTagName

Comment (RomanFedytskyi) [inline tests/testmicrodata.js]: reworked

**Comment by mshchygol** [comment]
LGTM

**Comment by vasylsemiliak** [comment]
LGTM

**Comment by Alexsv7** [comment]
LGTM

**Pull request merged**